docs/REST API/rest-api.md

```diff
@@ -6413,10 +6420,11 @@ Creates a script execution request and returns the execution identifier to retri
 
 #### Parameters
 
-| Name            | Type    | In   | Description                                      |
-| ----            | ------- | ---- | --------------------------------------------     |
-| host_id         | integer | body | **Required**. The host id to run the script on.  |
-| script_contents | string  | body | **Required**. The contents of the script to run. |
+| Name            | Type    | In   | Description                                                                                    |
+| ----            | ------- | ---- | --------------------------------------------                                                   |
+| host_id         | integer | body | **Required**. The host id to run the script on.                                                |
+| script_id       | integer | body | The script to run. The current contents of this saved script will be executed on the host. Only one of script_id and script_contents must be provided.  |
+| script_contents | string  | body | The contents of the script to run. Only one of script_id and script_contents must be provided. |
 
```

Comment:
Would add a note below the table clarifying what happens if you happen to include both `script_id` and `script_contents` (whether that's an error, or one being ignored). e.g.:
```suggestion
Note that if both `script_id` and `script_contents` are included in the request, this endpoint will respond with an error.
```
or

```suggestion
Note that if both `script_id` and `script_contents` are included in the request, `script_id` will be used and `script_contents` will be ignored.
```
(or whatever actually happens)